src/sentry/seer/code_review/tasks.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import orjson
+from django.conf import settings
+from urllib3.exceptions import HTTPError
+
+from sentry import options
+from sentry.net.http import connection_from_url
+from sentry.seer.signed_seer_api import make_signed_seer_api_request
+from sentry.silo.base import SiloMode
+from sentry.tasks.base import instrumented_task
+from sentry.taskworker.namespaces import seer_code_review_tasks
+from sentry.taskworker.retry import Retry
+from sentry.utils import metrics
+
+logger = logging.getLogger(__name__)
+
+
+# This needs to match the value defined in the Seer API:
+# https://github.com/getsentry/seer/blob/main/src/seer/routes/codegen.py
+SEER_CODEGEN_PATH = "/v1/automation/codegen"
+SEER_PR_REVIEW_RERUN_PATH = f"{SEER_CODEGEN_PATH}/rerun"
+PREFIX = "seer.code_review.check_run.rerun"
+
+
+@instrumented_task(
+    name="sentry.seer.code_review.tasks.process_github_webhook_event",
+    namespace=seer_code_review_tasks,
+    retry=Retry(times=3, delay=60),
+    silo_mode=SiloMode.REGION,
+    bind=True,  # Bind task instance to access self.request.retries for conditional logging
+)
+def process_github_webhook_event(self: Any, *, organization_id: int, **kwargs: Any) -> None:
+    """
+    Process GitHub webhook event by forwarding to Seer if applicable.
+
+    This will be expanded to handle other GitHub webhook events in the future.
+
+    Args:
+        self: Task instance (bound via bind=True)
+        organization_id: The organization ID (for logging/metrics)
+        **kwargs: Additional logging context & other parameters
+    """
+    original_run_id = kwargs.get("original_run_id")
+    if original_run_id is None:
+        logger.error("%s.error", PREFIX, extra=kwargs)
+        metrics.incr(f"{PREFIX}.outcome", tags={"status": "invalid_task_parameters"})
+        return
+
+    payload = {"original_run_id": original_run_id}
```

Comment:
This is the only value we need to send to Seer:
https://github.com/getsentry/seer/pull/4173/changes#diff-d022a5605fc7126215740d58900ce6e7c4be1899c6b4809c8e353bda0883a35dR143